iphone/cli/commands/_build.js

```diff
@@ -4945,6 +4945,13 @@ iOSBuilder.prototype.processTiSymbols = function processTiSymbols() {
 		if (this.useJSCore) {
 			contents += '\n#define USE_JSCORE_FRAMEWORK';
 		}
+		var infoPlist = this.infoPlist;
+		if (Array.isArray(infoPlist.UIBackgroundModes) && infoPlist.UIBackgroundModes.indexOf('remote-notification') !== -1) {
+			contents += '\n#define USE_TI_SILENTPUSH';
+		}
+		if (Array.isArray(infoPlist.UIBackgroundModes) && infoPlist.UIBackgroundModes.indexOf('fetch') !== -1) {
+			contents += '\n#define USE_TI_FETCH';
+		}
```

Comment:
But device builds were already addressed here at this line:
https://github.com/appcelerator/titanium_mobile/blob/master/iphone/cli/commands/_build.js#L4959